1223-curator_worklist/my_worklist_approval.feature

```diff
@@ -0,0 +1,55 @@
+Feature: Curator handles an Approval Request
+
+    In order to manage the amount of resources created
+    As an Editor (at a large Institution like NTNU)
+    I want to allow the User to publish resources
+
+	In order to reduce cost of training and secure high quality on all resources
+    As an Editor (at a small Institution Like NINA)
+    I want an Curator to approv all resources prior to publishing
+
+    In order to remove unnessesery work and frustration from duplicate resources 
+    As an Editor (at a medium Institution like SINTEF)
+    I want to allow Users to publish metadata resources, then a Curator approv the resource before the content is made publicly accessable
+
+    Rules:
+     - The User got publisher privilege
+     - User got publisher privilege, but all content need a Curator Approval to be publicly accessable
```

Comment:
User has publisher privilege, but all content needs Curator Approval to be publicly accessible